regression/validate-trace-xml-schema/validate-xsd/src/main/java/com/diffblue/validatexsd/Main.java

```diff
@@ -0,0 +1,50 @@
+package com.diffblue.validatexsd;
+
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXParseException;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Parameters;
+
+import javax.xml.XMLConstants;
+import javax.xml.transform.sax.SAXSource;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import javax.xml.validation.Validator;
+import java.io.FileInputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.concurrent.Callable;
+
+@Command(name = "validate-xsd", mixinStandardHelpOptions = true, version = "0.1",
```

Comment:
It makes me a little sad we weren't able to find an xsd validator that exists on apt, but there is not much code here anyway so I guess it's OK

Reply:
So after doing some more digging apparently there's xmllint? I think I can remove the java bits and just use that.

Reply:
Sounds good -> saves writing the tests! :tada: